gokart/build.py

```diff
@@ -16,12 +16,12 @@ def __init__(self, level: int = logging.CRITICAL):
         self.level = level
 
     def __enter__(self):
-        logging.disable(self.level)
+        logging.disable(self.level - 10)
         self.logger.setLevel(self.level)
         return self
 
     def __exit__(self, exception_type, exception_value, traceback):
-        logging.disable(self.default_level)
+        logging.disable(self.default_level - 10)
```

Comment:
Does it make sense to be 10? If not, use `sys.maxsize` to make the meaning clearer.

Reply:
@vaaaaanquish 
I think it should be 10,  because we want to change "disabling logs with level self.default_level and above" to "disabling logs with levels above self.default_level"

Reply:
I see. It seems like a good idea to put it in the comments.